backend/init_database.js

```diff
@@ -0,0 +1,58 @@
+const { debug } = require("console");
+const fs = require("fs");
+
+function addPoem (poem) {
+    const db = require('./database.json')
+    const newPoem = poem
+    db.push(newPoem)
+    fs.writeFile('database.json', JSON.stringify(db), err => {
+        if (err) throw err
+        console.log('Done writing') // Success
+    })
+}
+addPoem({'poem': 'this is a poem', 'date': 11242022, 'author': 'trisha','lat': 3, 'long': 4, 'route': 12 })
```

Comment:
once u've implemented the test script i explained above u can get rid of this line